datafusion/src/logical_plan/expr.rs

```diff
@@ -1317,6 +1319,83 @@ pub fn normalize_cols(
         .collect()
 }
 
+/// Rewrite sort on aggregate expressions to sort on the column of aggregate output
+#[inline]
```

Comment:
why would this need to be inlined? or can't the compiler infer that?

Reply:
Hmm, I saw there is `#[inline]` for `normalize_cols` above. `rewrite_sort_cols_by_aggs`'s looks similar (i.e., `into_iter().map().collect()`), so I added it too here. Should I remove it?

Reply:
> In Rust, a unit of (separate) compilation is a crate. If a function f is defined in a crate A, then all calls to f from within A can be inlined, as the compiler has full access to f. If, however, f is called from some downstream crate B, such calls can’t be inlined. B has access only to the signature of f, not its body.

So I think both of them don't need `#[inline]`.

Reply:
Ok, removed.